docs/book/src/topics/rosa/creating-a-cluster.md

```diff
@@ -1,30 +1,83 @@
 # Creating a ROSA cluster
 
 ## Permissions
-CAPA controller requires an API token in order to be able to provision ROSA clusters:
+### Authentication using service account credentials
+CAPA controller requires service account credentials to be able to provision ROSA clusters:
+1. Visit [https://console.redhat.com/iam/service-accounts](https://console.redhat.com/iam/service-accounts) and create a new service account.
 
-1. Visit [https://console.redhat.com/openshift/token](https://console.redhat.com/openshift/token) to retrieve your API authentication token
-
-1. Create a credentials secret within the target namespace with the token to be referenced later by `ROSAControlePlane`
+1. Create a new kubernetes secret with the service account credentials to be referenced later by `ROSAControlPlane`
     ```shell
     kubectl create secret generic rosa-creds-secret \
-      --from-literal=ocmToken='eyJhbGciOiJIUzI1NiIsI....' \
-      --from-literal=ocmApiUrl='https://api.openshift.com' 
+      --from-literal=ocmClientId='....' \
+      --from-literal=ocmClientSecret='eyJhbGciOiJIUzI1NiIsI....' \
+      --from-literal=ocmApiUrl='https://api.openshift.com'
     ```
 
-    Alternatively, you can edit CAPA controller deployment to provide the credentials:
+    Note: to consume the secret without the need to reference it from your `ROSAControlPlane`, name your secret as `rosa-creds-secret` and create it in the CAPA manager namespace (usually `capa-system`):
     ```shell
-    kubectl edit deployment -n capa-system capa-controller-manager
+    kubectl -n capa-system create secret generic rosa-creds-secret \
+      --from-literal=ocmClientId='....' \
+      --from-literal=ocmClientSecret='eyJhbGciOiJIUzI1NiIsI....' \
+      --from-literal=ocmApiUrl='https://api.openshift.com'
     ```
 
-    and add the following environment variables to the manager container:
-    ```yaml
-      env:
+
+### Authentication using SSO offline token (DEPRECATED)
+Instead of the service account credentials you can use SSO offline token, that you can specify either as a secret or specify the offline token
```

Comment:
```suggestion
The SSO offline token is been deprecated, it is recommended to use the service account as mentioned above. However, for old users you still can visit https://console.redhat.com/openshift/token to retrieve your SSO offline authentication token.
```

Reply:
I restructured the text a bit, but I'd rather not call any CAPI users "old". The instructions to go to consoleDot to retrieve the token are part of the text below.